dts/arm/nordic/nrf5.dtsi

```diff
@@ -0,0 +1,13 @@
+
+/ {
+	sw_pwm: sw-pwm {
+		compatible = "nordic,nrf-sw-pwm";
+		timer-instance = <0>;
+		timer-capture-compare-count = <0>;
```

Comment:
I don't see the point in specifying all properties here, especially with invalid values like this one. What's wrong in having only the common ones here (i.e. `compatible`, `status`, and `label`)?

Reply:
Not needed - removed.